package.json

```diff
@@ -34,14 +34,16 @@
     "@typescript-eslint/eslint-plugin": "^5.27.1",
     "@typescript-eslint/parser": "^5.27.1",
     "eslint": "^8.27.0",
+    "geckodriver": "^3.1.0",
     "mocha": "^10.1.0",
     "mockery": "^2.1.0",
-    "nightwatch": "^2.5.4",
+    "nightwatch": "^2.6.21",
     "nock": "^13.2.9",
     "typescript": "^4.7.3"
   },
   "dependencies": {
     "@nightwatch/mobile-helper": "^0.1.11",
+    "@nightwatch/setup-tools": "^0.0.2",
```

Comment:
I would suggest not to have a hard dependency on `@nightwatch/setup-tools` because whenever we update `@nightwatch/setup-tools` we would need to update `create-nightwatch` as well.

Also, if we could figure out a way to always use the latest version of `@nightwatch/setup-tools` in `create-nightwatch`, it would solve the problem of having to use `@latest` while running the tool, because `create-nightwatch` would automatically run the latest code.

Reply:
There is no way for us to use the latest version of `@nightwatch/setup-tools` the same caching mechanism in npm-cli prevents us from doing that. You are right on the version locking part.